zebra-state/src/tests.rs

```diff
@@ -15,6 +78,22 @@ static BLOCK_LOCATOR_CASES: &[(u32, u32)] = &[
     (10000, 9901),
 ];
 
+use proptest::prelude::*;
+
+#[test]
+fn round_trip_work_expanded() {
+    zebra_test::init();
+
+    proptest!(|(work in 1..std::u128::MAX)| {
+        let work: U256 = work.into();
```

Comment:
This is exactly what `Arbitrary for Work` does:
```suggestion
    proptest!(|(work in Any<Work>)| {
```

Also feel free to replace the current `Arbitrary for Work` implementation with your simpler implementation.
(And move `From<u128> for Work` to the arbitrary module.)
```
|(work in 1..std::u128::MAX)| {
    let work: U256 = work.into();
    work
}
```

Here's the existing implementation:
https://github.com/ZcashFoundation/zebra/blob/main/zebra-chain/src/work/difficulty/arbitrary.rs